OctopusDSC/OctopusDSCHelpers.ps1

```diff
@@ -57,7 +67,11 @@ function Request-File {
             Write-Verbose "Downloading $url to $saveAs"
 
             try {
-                Invoke-WebClient -Url $url -OutFile $saveAs
+                switch ($downloadMethod)
```

Comment:
I dont think we need this switch block - `WebClient` can "download" from a network share.

Reply:
If this is removed, I think we can combine the two `switch` statements into one - we wont need the `$downloadMethod` variable.